modules/caddyhttp/fileserver/browselisting.go

```diff
@@ -101,6 +101,9 @@ type browseListing struct {
 
 	// If ≠0 then Items have been limited to that many elements.
 	ItemsLimitedTo int
+
+	// If ≠0 then Items starting from that many elements.
+	ItemsFrom int
```

Comment:
```suggestion
	ItemOffset int
```

Reply:
Also changed the browse template file to 'offset'.